di_website/publications/models.py

```diff
@@ -252,7 +281,7 @@ def get_context(self, request):
         return context
 
     class Meta():
-        verbose_name = 'Publication Index Page'
+        verbose_name = 'Resources Page'
```

Comment:
I'd rather you keep the nomenclature ... `Resource Index Page`

Reply:
Updated to maintain naming convention